sklearn/utils/tests/test_stats.py

```diff
@@ -13,39 +13,39 @@ def test_weighted_percentile():
     y[50] = 1
     sw = np.ones(102, dtype=np.float64)
     sw[-1] = 0.0
-    score = _weighted_percentile(y, sw, 50)
-    assert approx(score) == 1
+    value = _weighted_percentile(y, sw, 50)
+    assert approx(value) == 1
```

Comment:
I am not opposed to this change (because "score" can cause confusion with the scoring functions of scikit-learn which are unrelated), however the name "score" was actually correct according to Wikipedia.

Maybe we could use `percentile_value` instead (or even `percentile_score`)?